src/_posts/platform/app/scaling/2000-01-01-scaling.md

```diff
@@ -0,0 +1,225 @@
+with predictable growth and resource needs.
+
+### Horizontal Scaling
+
+Horizontal scaling involves adding (*scaling out*) or removing (*scaling in*)
+instances of a resource to distribute the workload across all these instances.
+When using Scalingo, horizontal scaling is accomplished by either booting
+additional containers or by removing running ones.
+
+The platform [**automatically** routes the traffic]({% post_url platform/internals/2000-01-01-routing %}#requests-scheduling)
+to the available instances. This process, called *load-balancing*, allows to
+evenly distribute the load across multiple containers.
+
+Horizontal scaling has multiple advantages:
+- it adds fault tolerance and resilience to your application, as the failure of
+  one instance does not affect others.
+- it can scale almost infinitely, allowing your application to grow fast.
+- it can save costs, as the number of running containers can be automatically
+  adjusted (up and down) depending on your current application needs.
+
+For all these reasons, horizontal scaling is generally better suited for
+applications with distributed architectures that require high availability, and
+the ability to adapt to unpredictable or fluctuating workloads. It's often the
+best approach in PaaS environments such as Scalingo.
+
+### Chosing Between Vertical and Horizontal Scaling
+
+Both forms of scaling are complementary. You will often need to scale out/in
+(horizontal), and to scale up/down (vertical) your application to fit the
+evolution of the traffic of your app.
+
+Here is a quick comparison table, in the context of a Platform as a Service:
+
+|                 | Vertical Scaling                        | Horizontal Scaling                |
+| --------------- | --------------------------------------- | --------------------------------- |
+| **Approach**    | Enhancing individual instance capacity  | Adding more instances             |
+| **Cost**        | Can become expensive at higher limits   | Often more cost-efficient         |
+| **Resilience**  | Low (single point of failure)           | High (distributed resources)      |
+| **Flexibility** | Limited by physical/virtual constraints | High                              |
+| **When**        | Lack or overuse of CPU or RAM (swap increase or decrease) | Traffic increase or decrease, only if app is distributed |
+
+
+## Limitations
+
+- Vertical scaling is limited by the platform. The biggest container we can
+  currently boot is the `2XL` container, with 4GB of RAM. For a comprehensive
+  list of container sizes and corresponding specifications, please see our
+  [dedicated documentation page]({% post_url platform/internals/2000-01-01-container-sizes %}).
+- Horizontal scaling is limited by default to a maximum of 10 containers per
+  [process type]({% post_url platform/app/2000-01-01-procfile %}). This limit
+  can be increased via our support team.
+
+
+## Costs
```

Comment:
I find this section blurry. I'm not sure to understand what you want to say here. Could you explain where you want to go here?

Reply:
The goal is to make crystal-clear that scaling operations are free of charge, so users know they can use this feature for free.
But it also seemed important to me to precise at this point that scaling up or out induce a cost increase.
@benjaminach's update seems better. I propose to keep it for now.
We can still remove it later if it creates confusion :)